constants/extraRpcs.js

```diff
@@ -8220,20 +8220,6 @@ export const extraRpcs = {
   295: {
     rpcs: ["https://hedera.linkpool.pro"],
   },
-  2741: {
```

Comment:
I don't think this should be removed, but instead should be for Abstract Sepolia (chain ID 11124).